extensions/swagger-ui/deployment/src/main/java/io/quarkus/swaggerui/deployment/SwaggerUiProcessor.java

```diff
@@ -70,9 +70,9 @@ void feature(BuildProducer<FeatureBuildItem> feature) {
     @BuildStep
     @Record(ExecutionTime.STATIC_INIT)
     public void registerSwaggerUiServletExtension(SwaggerUiRecorder recorder,
```

Comment:
Super minor nit-pick: I think it would be good to rename the method now that servlet isn't involved